frontend/src/pages/CompareV2.tsx

```diff
@@ -233,9 +239,65 @@ function VisualizationPlaceholder(props: VisualizationPlaceholderProps) {
   );
 }
 
+interface VisualizationPanelItemProps {
+  metricsTab: MetricsType;
+  metricsTabText: string;
+  linkedArtifact: LinkedArtifact | undefined;
+  namespace: string | undefined;
+}
+
+function VisualizationPanelItem(props: VisualizationPanelItemProps) {
+  const { metricsTab, metricsTabText, linkedArtifact, namespace } = props;
+
+  const { data: viewerConfigs } = useQuery<ViewerConfig[], Error>(
+    [
+      'viewerConfig',
+      {
+        artifact: linkedArtifact?.artifact.getId(),
+        namespace,
+      },
+    ],
+    async () => {
+      let viewerConfigs: ViewerConfig[] = [];
+      if (linkedArtifact) {
+        try {
+          if (metricsTab === MetricsType.HTML) {
+            viewerConfigs = await getHtmlViewerConfig([linkedArtifact], namespace);
+          } else if (metricsTab === MetricsType.MARKDOWN) {
+            viewerConfigs = await getMarkdownViewerConfig([linkedArtifact], namespace);
+          }
+        } catch (err) {
+          throw err;
+        }
+      }
+      return viewerConfigs;
+    },
+    { staleTime: Infinity },
+  );
+
+  if (!linkedArtifact) {
+    return <VisualizationPlaceholder metricsTabText={metricsTabText} />;
+  }
+
+  if (metricsTab === MetricsType.CONFUSION_MATRIX) {
+    return (
+      <React.Fragment key={linkedArtifact.artifact.getId()}>
+        <ConfusionMatrixSection artifact={linkedArtifact.artifact} />
+      </React.Fragment>
+    );
+  }
+
+  // TODO(zpChris): Show loading and error pages for HTML and Markdown.
+  if (viewerConfigs && (metricsTab === MetricsType.HTML || metricsTab === MetricsType.MARKDOWN)) {
+    return <PlotCard configs={viewerConfigs} title={`Static ${metricsTabText}`} />;
```

Comment:
Question: why do we use `Static ` as prefix of title here?

Reply:
I only included it because this is the label on `MetricsVisualizations.tsx`: https://github.com/kubeflow/pipelines/blob/49c3587591fc291dc4b0f85ea7339d330ffe451c/frontend/src/components/viewers/MetricsVisualizations.tsx#L183-L195

I am happy to remove that label, I'm guessing it was included because the HTML and Markdown content is fixed (and not dynamic).

Reply:
I see, I think that all metrics are static by default. It is okay to keep it for now. If we receive feedback that it is not needed, we can remove it later.